### ISSUE TITLE
Wrong number in ID column in xlsx

### ISSUE THREAD

**Issue by zhangchi1006** [juliedwhite/MeshMonkValidation#1]
In your attached xlsx, the `ID` column has 35 unique IDs, rather than 41 unique IDs. For example `MM_125` has been replicated for 38 times rather than 19 times. Also, there is a missing directory called `Tables\` which should be created ahead, otherwise the txt results cannot be written to the disk properly.

Thanks for your wonderful work! It is still a little hard to install the MeshMonk on Ubuntu or Windows. A Docker image would be appreciated since I have thousands CBCT to manipulate:)

**Comment by zhangchi1006** [comment]
I found the same duplicate ID in the  `ID` column in your Supplementary Table S8.

**Comment by juliedwhite** [comment]
Hi. Thanks for pointing out these issues. 

I've updated the Rmd file to create the `Tables/` directory if it does not exist already in the working directory. I've also updated the landmarking excel sheet to reflect ID randomization without duplicates. The duplicate IDs did not reflect duplicate data but were accidentally incorporated when we randomized the ID numbers to ensure anonymity. I've also updated the manuscript SI with these new IDs and have added this and the manuscript PDF to the repo. Your best bet is probably to re-clone the repo to grab all of these.

As for the Docker image, you're welcome to post that as a feature request on the TheWebMonks/meshmonk repo. That's where we host the toolbox code and decisions on any updates would be made by the WebMonks team.
